Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.4.1"
 authors = [
   "Ralph Giles <giles@mozilla.com>",
   "Matthew Gregan <kinetik@flim.org>",
+  "Guillaume Gomez <guillaume1.gomez@gmail.com>"
```

Comment:
If you don't mind I think it makes more sense to keep the author's list to just the two primary contacts. Your contributions will be clear in the commit history.